Insights/IP/IP_TI_map.yaml

```diff
@@ -0,0 +1,214 @@
+SchemaVersion: '1.0'
+Type: KQL
+Provider: Sentinel
+DataTypes:
+  - DataType: Heartbeat
+  - DataType: VMConnection
+  - DataType: VMComputer
+  - DataType: ServiceMapComputer_CL
+  - DataType: WireData
+  - DataType: ProtectionStatus
+  - DataType: DeviceNetworkInfo
+  - DataType: DeviceNetworkEvents
+  - DataType: AzureNetworkAnalytics_CL
+  - DataType: DnsEvents
+  - DataType: CommonSecurityLog
+  - DataType: Event
+  - DataType: SecurityEvent
+  - DataType: Syslog
+EntitiesFilter: 
+ Host_OsFamily: []
+RequiredInputFieldsSets: 
+ - - IP_Address
+BaseQuery: |  
+  let GetIPStats = (IP_Address:string){
+  //checking time span to lock to 7 days or less for Entity page usage
+  let start = datetime('{{Start_Time_ISO}}');
+  let end = datetime('{{End_Time_ISO}}');
+  let end_start = datetime_diff('day',end,start);
+  let start_time = iff(end_start > 7, end - 7d, start);
+  let end_time = end;
+  let IpStats = (union isfuzzy=true
+  (
+  VMConnection
+  | where TimeGenerated between (start_time..end_time)
+  | where SourceIp =~ IP_Address
+  | where SourceIp != DestinationIp
+  | where Direction =~ "outbound"
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), OutboundCount = countif(Direction =~ "outbound") by IPAddress = SourceIp, Type, RemoteIPAddress = DestinationIp, Direction, SentBytes = tolong(BytesSent), ReceivedBytes = tolong(BytesReceived)
```

Comment:
It looks like "| summarize .. by ... SentBytes = tolong(BytesSent), ReceivedBytes = tolong(BytesReceived)" will generate a row per each unique row that SentBytes or Received.  How will this behave if there are multiple connections to the same IP address? Does it make sense to include this in the aggregation part of the summarize? e.g., SentBytes = sum(tolong(BytesSent)), ReceivedBytes = sum(tolong(BytesSent))